src/components/ui/input-password.tsx

```diff
@@ -22,6 +22,7 @@ function PasswordInput({
       />
       <button
         type="button"
+        tabIndex={-1}
```

Comment:
this change is not allowed. if you think there's an issue with shadcn's component, file an issue there instead.

https://github.com/shadcn-ui/ui